src/utils.js

```diff
@@ -1,9 +1,37 @@
-import { DayConverter } from "./components/FeatureViews/Accounts/TabComponents/CourseSessionStatus";
+import {DayConverter} from "./components/FeatureViews/Accounts/TabComponents/EnrollmentView";
+import {useDispatch, useSelector} from "react-redux";
+import React, {useMemo, useState} from "react";
+import {useHistory} from "react-router-dom";
+import {bindActionCreators} from "redux";
+import * as registrationActions from "./actions/registrationActions";
+import Button from "@material-ui/core/Button";
+import DialogTitle from "@material-ui/core/es/DialogTitle";
+import DialogContent from "@material-ui/core/es/DialogContent";
+import DialogContentText from "@material-ui/core/es/DialogContentText";
+import DialogActions from "@material-ui/core/DialogActions";
+import Dialog from "@material-ui/core/es/Dialog";
+import MenuItem from "@material-ui/core/MenuItem";
+import PropTypes from 'prop-types';
 
-const timeOptions = {
+export const timeFormat = {
+    "hour12": false,
+    "hour": "2-digit",
+    "minute": "2-digit",
+};
+
+export const dateFormat = {
+    "year": "numeric",
+    "month": "2-digit",
+    "day": "2-digit",
+};
+
+export const dateTimeFormat = {
+    "year": "numeric",
+    "month": "2-digit",
+    "day": "2-digit",
+    "hour12": false,
     "hour": "2-digit",
     "minute": "2-digit",
-    "hour12": true,
 };
```

Comment:
Write this simply as:

    {
        ...dateFormat,
        ...timeFormat,
    }